ggshield/verticals/secret/secret_scan_collection.py

```diff
@@ -70,6 +77,18 @@ def censor(self) -> None:
     def has_policy_breaks(self) -> bool:
         return self.scan.has_policy_breaks
 
+    def policy_breaks(
+        self, *, diff_kinds: Optional[List[DiffKind]] = None
```

Comment:
Would be nicer if diff_kinds was a set instead of a list, since we don't want to have duplicate kinds.